fluent.runtime/CHANGELOG.rst

```diff
@@ -0,0 +1,9 @@
+Changelog
+=========
+
+fluent.runtime 0.1 (January 21, 2019)
+-------------------------------------
+
+First release to PyPI of ``fluent.runtime``. This release contains a
+``FluentBundle`` implementation that can generate translations from FTL spec. It
+targets the Fluent 0.6 spec.
```

Comment:
I think it actually supports Syntax 0.7 :) [`setup.py` lists `fluent>=0.9,<0.10`](https://github.com/projectfluent/python-fluent/blob/fluent.runtime%400.1/fluent.runtime/setup.py#L29), and `fluent` 0.9 [added parsing support for Syntax 0.7](https://github.com/projectfluent/python-fluent/releases/tag/0.9.0).

Also, can you link to the Spec release page at https://github.com/projectfluent/fluent/releases/tag/v0.7.0, please?